src/main/java/org/humancellatlas/ingest/client/IngestApiClient.java

```diff
@@ -111,7 +111,7 @@ public SubmissionEnvelopeReference envelopeReferencesFromEnvelopeId(String envel
 
     public SubmissionEnvelopeReference referenceForSubmissionEnvelope(SubmissionEnvelopeMessage message) {
         return new SubmissionEnvelopeReference(message.getDocumentId(),
-                message.getDocumentUuid(), "",
+                message.getDocumentUuid(), null,
```

Comment:
`null` param is as evil as the empty string.
Either create another constructor that does not include this argument, a factory method named appropriately.
It is difficult to review since I don't know what the null is, without reading more code.

Reply:
Used AllArgsConstructor and another constructor which doesn't require the state.